e2e/node/basic/mainnet.test.ts

```diff
@@ -168,20 +161,3 @@ describe('controllers', () => {
   });
 });
 
-describe('bitcoin query', async () => {
```

Comment:
would a test that tests that the bitcoin queries are stripped out of the IDL be a valuable addition?